tests/core/test_aspects.py

```
@@ -1154,6 +1154,255 @@ def test_chart_data_natal_applies_luminary_bonus(self, _subject):
+        assert not has_aspect_keyed_adjustments({"Sun": 1.5})
+        assert not has_aspect_keyed_adjustments({})
+        assert not has_aspect_keyed_adjustments(None)
+
+
+class TestAspectKeyedExtraOrbInAspectDetection:
+    """get_aspect_from_two_points with a per-aspect extra_orb mapping."""
+
+    SETTINGS = [
+        {"name": "conjunction", "degree": 0, "orb": 6.0},
+        {"name": "trine", "degree": 120, "orb": 6.0},
+    ]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)